src/api/loader.coffee

```diff
@@ -41,22 +41,27 @@ getResponseDataByEnv = (isLocal, eventData, response) ->
 
 
 handleAPIEvent = (apiEventChannel, baseUrl, setting, eventData = {}) ->
+
   isLocal = setting.loadLocally
   # simulate server delay
   delay = if isLocal then 200 else 0
 
   apiSetting = getAjaxSettingsByEnv(isLocal, baseUrl, setting, eventData)
+  if apiSetting.method is 'GET'
```

Comment:
Yeah, i wasn't 100% sure about only GET but figured that if we attempted to prevent POST,etc then we'd have to check the body since you could legit want to post multiple things to the same url. 

I can't see why you'd ever want to call the same GET concurrently though.